Dockerfile

```diff
@@ -1,10 +1,11 @@
-FROM registry.fedoraproject.org/fedora:28
+FROM registry.fedoraproject.org/fedora:29
 WORKDIR /root/containerbuild
 
 # Only need a few of our scripts for the first few steps
 COPY ./build.sh ./deps.txt ./vmdeps.txt ./build-deps.txt /root/containerbuild/
 RUN ./build.sh configure_yum_repos
 RUN ./build.sh install_rpms
+RUN dnf install -y https://kojipkgs.fedoraproject.org//packages/virt-manager/2.1.0/1.fc29/noarch/virt-install-2.1.0-1.fc29.noarch.rpm https://kojipkgs.fedoraproject.org//packages/virt-manager/2.1.0/1.fc29/noarch/virt-manager-common-2.1.0-1.fc29.noarch.rpm
```

Comment:
will remove this once virt-install hits testing repo: https://bodhi.fedoraproject.org/updates/FEDORA-2019-c38a307cd5

Reply:
ok removed this since the update hit testing repos